README.md

```diff
@@ -183,6 +183,19 @@ $ npm install
 $ grunt
 ```
 
+### Elastic Search workflow
+Elastic Search needs to be running to display wordpress content:
+1. Use the sheer virtualenv:
+    ```bash
```

Comment:
When I look at the readme in the browser, step 1 appears on the same line as the previous sentence. Does there need to be a new line between them?